priv/repo/seeds.exs

```diff
@@ -1,11 +1,4 @@
-# Script for populating the database. You can run it as:
-#
-#     mix run priv/repo/seeds.exs
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     CoursePlanner.Repo.insert!(%CoursePlanner.SomeModel{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+CoursePlanner.Repo.delete_all CoursePlanner.User
```

Comment:
I am note sure what is a good practice for seeds, but we can have `priv/repo/prod_seeds.exs` and `priv/repo/dev_seeds.exs`, then we interpolate the environment into the file name when we define the `ecto.setup` alias